From stable-bounces@linux.kernel.org Mon Dec 17 17:18:52 2007
From: Matthew Wilcox <matthew@wil.cx>
Date: Tue, 18 Dec 2007 00:44:43 +0100
Subject: m68k: Export cachectl.h
To: stable@kernel.org
Cc: Matthew Wilcox <matthew@wil.cx>
Message-ID: <20071217234443.GM29139@stro.at>
Content-Disposition: inline

From: Matthew Wilcox <matthew@wil.cx>

patch e92042e5c009d84ba741ec4a978a13f260e6ee24 in mainline.

m68k: Export cachectl.h

libffi in GCC 4.2 needs cachectl.h to do its cache flushing.  But we
don't currently export it.  I believe this patch should do the trick.

Signed-off-by: Matthew Wilcox <matthew@wil.cx>
Cc: maximilian attems <max@stro.at>
Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/asm-m68k/Kbuild |    1 +
 1 file changed, 1 insertion(+)

--- a/include/asm-m68k/Kbuild
+++ b/include/asm-m68k/Kbuild
@@ -1 +1,2 @@
 include include/asm-generic/Kbuild.asm
+header-y += cachectl.h
